traefikee/Chart.yaml

```
@@ -8,7 +8,7 @@ icon: https://doc.traefik.io/traefik-enterprise/assets/images/logo-traefik-enter
 # Because of https://github.com/helm/helm/issues/3810 the pre-release version suffix has to be define.
 # This allows the installation on Kubernetes cluster with a pre-release version (e.g. v1.19.9-gke.1900)
 kubeVersion: ">= 1.14.0-0"
-version: 3.0.2
+version: 3.0.3
```

```suggestion
version: 3.0.2
```
You should not update the chart version in this PR.

---

indeed, the chore will create the release after, I forgot